### ISSUE TITLE
Implement Extension API endpoints

### ISSUE THREAD

**Issue by billygerhard** [twurple/twurple#164]
## Feature Request

Twitch provides APIs for extensions under the https://api.twitch.tv/helix/extensions/ namespace.

https://dev.twitch.tv/docs/api/reference#get-extension-configuration-segment

- [ ] Get Live Channels with Extension Activated
- [ ] Get Released Extensions
- [ ] Get Extension Bits Products
- [ ] Update Extension Bits Product

Postponed (see comment below):

- Get Extension
- Create Extension Secret
- Get Extension Secret
- Set Extension Required Configuration
- Set Extension Configuration Segment
- Get Extension Configuration Segment
- Send Extension PubSub Message
- Send Extension Chat Message

**Comment by d-fischer** [comment]
These endpoints were just deprecated: https://discuss.dev.twitch.tv/t/how-extensions-are-affected-by-the-legacy-twitch-api-v5-shutdown/32708

I will implement the corresponding Helix endpoints though.

**Comment by d-fischer** [comment]
This will be done in two parts:

- The endpoints that require an OAuth token will be added for 5.0.
- The endpoints that require a JWT will be added somewhere down the line in 5.x, but not in 5.0. I will create a followup issue for this part once the first part is done.

**Comment by billygerhard** [comment]
I updated the packages, and I am still getting the same thing

There are 58 requests I am trying to process, and it is pulling 17 of them, and 39 are staying in the queue
```
[2021-09-06T16:12:42.718Z] 🐞 rate-limiter DEBUG Remaining requests: 0
NodeLogger.mjs:99 [2021-09-06T16:12:42.718Z] 🐞 rate-limiter DEBUG Doing 1 requests, new queue length is 33
NodeLogger.mjs:99 [2021-09-06T16:12:42.942Z] 🐞 rate-limiter DEBUG Remaining requests: 0
NodeLogger.mjs:99 [2021-09-06T16:12:42.942Z] 🐞 rate-limiter DEBUG Doing 1 requests, new queue length is 35
NodeLogger.mjs:99 [2021-09-06T16:12:43.171Z] 🐞 rate-limiter DEBUG Remaining requests: 0
NodeLogger.mjs:99 [2021-09-06T16:12:43.172Z] 🐞 rate-limiter DEBUG Doing 1 requests, new queue length is 37
NodeLogger.mjs:99 [2021-09-06T16:12:43.398Z] 🐞 rate-limiter DEBUG Remaining requests: 0
NodeLogger.mjs:99 [2021-09-06T16:12:43.398Z] 🐞 rate-limiter DEBUG Doing 1 requests, new queue length is 39
```

The function I am running. I have it inside a bigger class, so I took out some shared globals, but it should work. I have my twitch client on a global, not as a parameter like I put below, but makes it easier to reproduce.
```
async getFollowers(userName:string, twitchClient:ApiClient){
    let users = [];
    let userFollowCounts = {};
    let userFollowCountProcessed = 0;
    const channel:HelixUser = await twitchClient.users.getUserByName(userName);
    let userID = channel.id;
    let request = twitchClient.users.getFollowsPaginated({followedUser:channel});
    let followCount = await request.getTotalCount();
    let follows = await request.getAll();
    let userIds = follows.map(follow=>follow.userId);
    users = await twitchClient.users.getUsersByIds(userIds);
    for(let user of users){
     twitchClient.users.getFollowsPaginated({followedUser:user}).getTotalCount()
        .then((followCount)=>{
          userFollowCounts[user.id] = followCount;
          userFollowCountProcessed++;
        });
    }
  }
```
